azext_iot/iothub/params.py

```diff
@@ -246,3 +246,81 @@ def load_iothub_arguments(self, _):
             options_list=["--content-type", "--ct"],
             help="MIME Type of file.",
         )
+
+    with self.argument_context("iot edge hierarchy") as context:
+        context.argument(
+            "devices",
+            options_list=["--device", "-d"],
+            nargs="+",
+            action="append",
+            help="Space-separated key=value pairs corresponding to properties of the edge device to create. "
+            "--device can be used 1 or more times. Review help examples for parameter usage."
+        )
+        context.argument(
+            "clean",
+            options_list=["--clean", "-c"],
+            arg_type=get_three_state_flag(),
+            help="Deletes all devices in target hub before creating new hierarchy.",
+        )
+        context.argument(
+            "visualize",
+            options_list=["--visualize", "--vis", "-v"],
+            arg_type=get_three_state_flag(),
+            help="Shows visualizations of device hierarchy and progress of various tasks "
+            "(device creation, setting parents, updating configs, etc).",
+        )
+        context.argument(
+            "config_file",
+            options_list=["--config-file", "--config", "--cfg"],
+            help="Path to device hierarchy config file",
+        )
+        context.argument(
+            "device_auth_type",
+            arg_type=get_enum_type(
+                [DeviceAuthType.shared_private_key, DeviceAuthType.x509_ca]
+            ),
+            options_list=["--device-auth-type", "--device-auth"],
+            help="Device to hub authorization mechanism.",
+        )
+        context.argument(
+            "default_edge_agent",
+            options_list=["--default-edge-agent", "--default-agent", "--dea"],
+            help="Default edge agent for created devices.",
+        )
+        context.argument(
+            "device_config_template",
+            options_list=["--device-template", "--dt"],
+            help="Path to config.toml file to use as a basis for edge device configs.",
+        )
+        context.argument(
+            "bundle_output_path",
+            options_list=[
+                "--output-path",
+                "--out",
+            ],
+            help="Directory path to output device configuration bundles.",
+        )
+        context.argument(
```

Comment:
Any opportunities for organizing related arguments as arg groups? For example an arg group for cert related args.